editor/editor_inspector.cpp

```diff
@@ -4095,6 +4095,7 @@ void EditorInspector::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_READY: {
+			ERR_FAIL_NULL(EditorFeatureProfileManager::get_singleton());
```

Comment:
Now that CI testing runs through this, and we're not fully setting up the editor, we need to exit here as the `EditorFeatureProfileManager` singleton does not get constructed.